src/main/java/Task.java

```diff
@@ -0,0 +1,116 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    protected static int taskIndex = 0;
+    protected static final int MAX_ARRAY_SIZE = 100;
+    protected static Task[] taskArray = new Task[MAX_ARRAY_SIZE];
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public static void printList() {
+        int index = 0;
+        printTaskHeader();
+        for(Task task: taskArray) {
+            // Empty array
+            if(index >= taskIndex){
+                break;
+            }
```

Comment:
I like how you put curly bracket although there is only one statement inside. I see this in other places as well.